### ISSUE TITLE
Prediction function border1

### ISSUE THREAD

**Issue by DStone79** [zhouhaoyi/Informer2020#269]
Please excuse me if I have misunderstood, but shouldn't the border1:border2 length in the prediction Dataset be 120 rather than 96?

**Comment by cookieminions** [comment]
`border1:border2` is the length of train set, valid set or test set.

**Comment by DStone79** [comment]
Yes, but it is also in the prediction dataloader, where it is set to len(df_raw) - sequence length to len(df_raw) i.e. 96 in the standard configuration

Whereas to incorporate seq_len,label_len and pre_len it should be 120 ( 96 -48 +48 +24) shouldn't it?

**Comment by zhouhaoyi** [comment]
We keep the border consistent during training and testing. But you can modify that if necessary.
